src/Translator.java

```diff
@@ -0,0 +1,27 @@
+import java.util.HashMap;
+
+public class Translator {
+    private HashMap<Integer,String> numericAlpha;
```

Comment:
Andere mogelijkheid voor deze declaratie is: Map<Integer, String> numericAlpha = new HashMap<>(); En dan kan je regel 7 weg laten. Dat moet vervolgens ook werken :).